.github/workflows/release.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         run: pip install meson ninja
       - name: Install dependencies
         run: |
-          sudo apt-get install -y libudev-dev libxkbcommon-dev libdrm-dev libgbm-dev libegl1-mesa-dev libgles-dev libpango1.0-dev libsystemd-dev
+          sudo apt-get install -y check libudev-dev libxkbcommon-dev libdrm-dev libgbm-dev libegl1-mesa-dev libgles-dev libpango1.0-dev libsystemd-dev
```

Comment:
I suppose it would be helpful to actually run the test, but I'm not sure where to add that.

Reply:
Removing `--no-tests` in L41 will run tests. The bigger problem is that the existing ones aren't really meant to be unit tests and they require human inputs. Probably better to skip them in tests for now and at least enable testing on CI.

Reply:
I've removed the `test()` invocation for the existing tests in `tests/meson.build`, so only the new unit test is run and testing is enabled on CI.